zubbi/models.py

```diff
@@ -47,6 +47,41 @@
     "description": "description^1.2",
 }
 
+RESERVED_CHARACTERS = [
+    "+",
+    "-",
+    "=",
+    ">",
+    "<",
+    "!",
+    "(",
+    ")",
+    "{",
+    "}",
+    "[",
+    "]",
+    "^",
+    '"',
+    "~",
+    "*",
+    "?",
+    ":",
+    "\\",
+    "/",
+    # Elasticsearch also uses the following character sequences as
+    # reserved characters, but a translation table can only contain
+    # single characters. Since we haven't use them so far, I think
+    # we can live without them for now. In case this becomes a
+    # problem, we might want to escape them in a second replacement
+    # step without using a translation table.
+    # "&&",
+    # "||",
+]
+
+# Build a translation table from the list of reserved characters. This
+# will prefix each character in the list with a '/'.
```

Comment:
nit: It's using the backslash as escape char not the forwardslash.

Reply:
Thanks for the hint. I updated the comment.